static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -405,6 +449,21 @@ const StacktraceLinkWrapper = styled('div')`
   padding: 0 ${space(1)};
 `;
 
+const CopyButton = styled('button')`
```

Comment:
Rather than the custom `CopyButton`, would `<Button size="zero" priority="transparent">` have the same effect?